pulsar-common/src/main/java/org/apache/pulsar/common/policies/data/PolicyHierarchyValue.java

```diff
@@ -36,6 +36,8 @@
     @Getter
     private volatile T namespaceValue;
 
+    @VisibleForTesting
```

Comment:
Now that there is a getter, why do we need VisibleForTesting?

Reply:
Removed.